packages/xchain-solana/src/utils.ts

```diff
@@ -0,0 +1,12 @@
+import { Cluster } from '@solana/web3.js'
+import { Network } from '@xchainjs/xchain-client'
+
+export const getSolanaNetwork = (network: Network): Cluster => {
```

Comment:
I did't test it, but WDYT about using a literal type here?
Something like this:
```typescript
export const getSolanaNetwork = (network: Network): Cluster => {
  const networkMap: { [key in Network]: Cluster } = {
    [Network.Mainnet]: 'mainnet-beta',
    [Network.Stagenet]: 'mainnet-beta',
    [Network.Testnet]: 'testnet',
  };

  return networkMap[network];
}
```

Reply:
Good approach 😎